joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 10
arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
  state_publish_rate: 10
  action_monitor_rate: 10
  gains:
    joint1: { p: 1000.0, d: 10.0, i: 1.0, i_clamp: 1.0 }
    joint2: { p: 1000.0, d: 10.0, i: 1.0, i_clamp: 1.0 }
    joint3: { p: 1000.0, d: 10.0, i: 1.0, i_clamp: 1.0 }
    joint4: { p: 1000.0, d: 10.0, i: 1.0, i_clamp: 1.0 }
    joint5: { p: 1000.0, d: 10.0, i: 1.0, i_clamp: 1.0 }
    joint6: { p: 1000.0, d: 10.0, i: 1.0, i_clamp: 1.0 }
